packages/nimble-components/src/table/index.ts

```diff
@@ -117,6 +165,9 @@ export class Table<
             };
             return rowState;
         });
+        if (this.isConnected) {
```

Comment:
We should use `this.$fastController.isConnected` instead, it's aware of when bindings, etc in fast are ready.

It looks like `cell/index.ts` is another spot using `this.isConnected` instead of `this.$fastController.isConnected`, I wonder if it's related to the Safari issue. Either way we should update this spot

Reply:
Updated